apps/netutils/libcoap/LICENSE.GPL

```diff
@@ -0,0 +1,280 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
We can choose the revised BSD license. 
Please check my e-mail and add the license contents in each source file.
Thanks.